### DISCUSSION TITLE
How to solve "<MilvusException: (code=1, message=code: CollectionNotExists, reason: can't find collection: 444699607489585828)>"

### DISCUSSION THREAD

**Discussion by zhoukewei9700** [milvus-io/milvus#30202] [Q&A and General discussion]
I want to run the example of image similarity search, but I met this exception when inserting vectors into collection. I have already create the collection before.
![image](https://github.com/milvus-io/milvus/assets/77055544/10ec65d0-3dcd-4644-a059-e97b9dfbe3eb)
the code i use was written below:

`p_embed = (
    pipe.input('src')
        .flat_map('src', 'img_path', load_image)
        .map('img_path', 'img', ops.image_decode())
        .map('img', 'vec', ops.image_embedding.timm(model_name=MODEL, device=DEVICE))
)

connections.connect(host=HOST, port=PORT, user=USER, password=PASSWORD)
collection=Collection(COLLECTION_NAME)

p_insert = (
        p_embed.map(('img_path', 'vec'), 'mr', ops.ann_insert.milvus_client(
                    host=HOST,
                    port=PORT,
                    user=USER,
                    password=PASSWORD,
                    collection_name=COLLECTION_NAME,
                    ))
          .output('mr')
)

p_insert(INSERT_SRC)
print('Number of data inserted:', collection.num_entities)`

the Milvus version is 2.2.10, pymilvus version is 2.2.15, towhee version is 1.1.3

**Comment by yhmo** [comment]
Possible reasons: 
- the USER has no privilege to access the collection
- the collection is dropped by another client
- the connection connects to a specific database and the collection doesn't in the database

**Reply by zhoukewei9700** [reply]
thanks for helping, but I checked all those reasons and didn't find why.
the USER I use is root.
the collection exisits.
I only have the default database.

**Reply by zhoukewei9700** [reply]
It seems that the collection_id is not the same. I use `collection.describe()` and it shows the collection_id is 444699607492989410, but the exception shows the unfind collection_id is 444699607489585828

**Reply by yhmo** [reply]
A collection with the same name is deleted before? Or is there an alias created for the collection?
The server-side log might help to know what happened inside the server.

**Reply by zhoukewei9700** [reply]
Yes, there is a collection with the same name was deleted before, the problem was solved when I changed the COLLECTION_NAME. Really appreciate for your help.
